src/include/storage/store/column_chunk.h

```diff
@@ -151,12 +151,12 @@ class BoolColumnChunk : public ColumnChunk {
               enableCompression, hasNullChunk) {}
 
     void append(common::ValueVector* vector) final;
+    void appendOne(common::ValueVector* vector, common::vector_idx_t pos) final;
```

Comment:
This is fine temporarily, but we should refactor the two interfaces `append(common::ValueVector* vector)` and `appendOne(common::ValueVector* vector, common::vector_idx_t pos)` into a single one by passing SelVector as an argument,

```
void append(common::ValueVector* vector, SelVector& sel);
```